ivy_tests/test_ivy/helpers.py

```diff
@@ -1857,24 +1864,33 @@ def test_frontend_function(
 
     ret = frontend_fn(*args, **kwargs)
     ret = ivy.array(ret) if with_out and not ivy.is_array(ret) else ret
-    # assert idx of return if the idx of the out array provided
     out = ret
+    assert not with_out or not with_inplace
```

Comment:
please add a message here: `assert not with_out or not with_inplace, "only one of with_out or with_inplace can be set as True"`